src/main/java/com/cloudogu/smeagol/wiki/infrastructure/GitClient.java

```diff
@@ -177,6 +186,11 @@ private void pullChanges() throws GitAPIException, IOException {
         LOG.debug("pull changes from remote for repository {}", repository);
         ObjectId oldHead = git.getRepository().resolve("HEAD^{tree}");
 
+        // We have to be sure that the origin url points to the actual wiki repository url.
+        // This can be differ if the fqdn of the ecosystem has changed or after a migration
+        // from scm-manager v1 to v2.
+        ensureOriginMatchesRemoteUrl(git);
+
         git.pull()
                 .setRemote("origin")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "origin" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1192)